docs/get_started/product_overview/role_based_access.md

```diff
@@ -6,20 +6,20 @@ tags:
 hide: tags
 ---
 
-# Role Based Access Control
+# Role based access control
 
-HCL DX has rich features for controlling what users can see and do in the experiences created with the product. This ranges from simple access control for websites to granular control on content items.
+HCL DX has rich features for controlling what users can see and do in the experiences you create with the product. This control ranges from simple access control for websites to granular control on content items.
 
-## Access Control
+## Access control
 
-HCL DX provides a rich framework for managing authorizations to resources hosted in and served by the platform. This includes granular control over who can access what, and a rich set of possible extensions to integrate to your security environment.
+HCL DX provides a framework for managing authorizations to resources hosted in and served by the platform. This framework includes granular control over who can access what and a rich set of possible extensions to integrate with your security environment.
 
-Access can be controlled to pages, individual applications, content libraries, content items, asset collections and a host of other artifacts across the platform. Access is, by default, hierarchical meaning that access granted to a parent resource also applies to its children.
+Access can be controlled to pages, individual applications, content libraries, content items, asset collections and many other artifacts on the platform. Access is, by default, hierarchical, meaning that access granted to a parent resource also applies to its child resources.
 
-HCL DX provides a framework called "step up authentication" which allows the platform to manage multiple levels of authentication. For example, most of a site could be secured using simple user name and password authentication, but sensitive areas could require additional credentials.
+HCL DX provides a framework called "step up authentication," the platform uses to manage multiple levels of authentication. For example, most of a site might be secured by using simple user-name and password authentication but sensitive areas might require additional credentials.
```

Comment:
Shouldn't this be just "username"? No need for hyphen
```suggestion
HCL DX provides a framework called "step up authentication," which the platform uses to manage multiple levels of authentication. For example, most of a site might be secured by using simple username and password authentication but sensitive areas might require additional credentials.
```

Reply:
user name two words unless it's in code as a variable. here "user-name" is a compound adjective, so the two words are hyphenate to show that relationship.

"which the platform uses" OK!